Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG IMAGE_BACKEND=scratch
 ARG IMAGE_FRONTEND=scratch
 
 # Backend dependencies
-FROM eclipse-temurin:17-jdk-alpine AS geoviite-backend-dependencies
+FROM eclipse-temurin:17-jdk AS geoviite-backend-dependencies
```

Comment:
-alpine kuvaa ei ole saatavilla temurinista

Reply:
Jännä. Katselin että tuollainen kuitenkin olisi: jdk-ubi9-minimal
En ole ubi-imageita koskaan käyttänyt mutta ilmeisesti "The Universal Base Image Minimal is a stripped down image that uses microdnf as a package manager. This base image is freely redistributable" eli tuo voisi olla alpinemainen tarjonta red hatin puolelta. Voisi ehkä tutustua jossain kohtaa.

Reply:
Tässäpä olikin muuten jokin ajatuskatkos alkuperäisessä kommentissa, tuostahan siis nimenomaan on olemassa alpine-kuva, muttei sitä alpine-kuvaa ole saatavilla ARMille 😄

Reply:
Joo, en edes sisälukenut kommenttia oikein, oletin että tarkoitit juurikin tuota.